### PR TITLE
metal rods can mine through rock [QoL]

### PR DESCRIPTION
**Pull request by ghost**
[Changelogs]: # metal rods can now mine through lavaland rock

:cl: zamolxius
tweak: rods can mine with toolspeed 5
balance: metal rods can mine
code: made a new modularized rods.dmi and changed hippiestation.dme
refactor: made metal rods able to mine
/:cl:

[why]:  nothing else to do if you're stuck on mining. Metal is everywhere, you can just break a chair or something. I originally planned on having crowbars be able to mine but I fear them possibly conflicting with the crowbar tool type, plus i dont think i can do separate tool speeds for both usages or something
this is very slow compared to a pickaxe; you're just gouging the rock with a metal rod; open to tweaks and rebalances

### CODE REVIEW

File: hippiestation.dme [hunk 1]
```diff
@@ -2657,8 +2657,8 @@
 #include "hippiestation\code\game\gamemodes\messiah\MakeJesus.dm"
 #include "hippiestation\code\game\gamemodes\messiah\miracles.dm"
 #include "hippiestation\code\game\gamemodes\messiah\outfit.dm"
-#include "hippiestation\code\game\gamemodes\nuclear\nuclear.dm"
```
Comment (ghost) [inline hippiestation.dme:2660]: what

Comment (Manly-Dwarf) [reply hippiestation.dme:2660]: Ignore that.

**Comment by ghost** [comment]
bump please review

**Comment by Carbonhell** [comment]
shrug sure why not +1

**Review by ghost — APPROVED** [review]
looks okay

**Comment by Pyko1** [comment]
sure +1

Even tho what if ore walls would just have health, so you could use anything that deals damage to "mine". And mining tools would be just far more efficient usually.

**Comment by ghost** [comment]
yeah thatd be awesome

**Comment by earthmoonjune** [comment]
good with me +1

**Comment by ghost** [comment]
bump

**Pull request merged**